# Scope

## Scope of Rusteomics

Rusteomics is dedicated to advancing mass spectrometry-based omics data analysis through efficient data structures and algorithms tailored to the field. Our focus encompasses all aspects of mass spectrometry-based omics, including but not limited to data input/output and conversion, analyte characterization, identification and quantification.

All code within Rusteomics is developed in Rust or leverages Rust bindings to other programming languages. Contributions should be relevant to the broader scientific community and adhere to established community standards, such as those outlined by [HUPO-PSI](https://psidev.info/).

While Rusteomics initially focuses on proteomics, we actively welcome contributions in other omics fields, including metabolomics and nucleomics. A core principle is to ensure interoperability and reusability across different omics domains, promoting cross-disciplinary integration.

## Related Projects

Several complementary projects exist in related fields, each contributing valuable tools to the broader omics and bioinformatics ecosystem:

- [noodles](https://github.com/zaeleus/noodles) – Specializing in genomics & NGS formats, structured as a modular workspace of interdependent Rust crates.
- [rust-bio](https://github.com/rust-bio/rust-bio) – A comprehensive Rust library for sequence analysis, covering classical sequence-based statistical models and data structures.
- [pdbtbx](https://github.com/douweschulte/pdbtbx) – Focused on 3D protein structures and associated formats.
